src/details/owner_borrow.yaml

```diff
@@ -1,5 +1,405 @@
+- title: Understanding ownership
+  shortTitle: ownership
+  key: ownership
+  details: |
+    - Ownership is Rust’s most unique feature, and it enables Rust to make memory safety guarantees without needing a garbage collector.
+    - Rust’s central feature is ownership. All programs have to manage the way they use a computer’s memory while running.
+    - Some languages have garbage collection that constantly looks for no longer used memory as the program runs; in other languages, the programmer must explicitly allocate and free the memory.
+    - Rust uses a third approach: memory is managed through a system of ownership with a set of rules that the compiler checks at compile time.
+    - None of the ownership features slow down your program while it’s running.
+    - The Rust compiler checks if a program obeys the ownership rules at compile time. If the program follows these rules, it can run. If it doesn’t, the compiler refuses to produce an executable.
+    - Rust verifies the ownership rules using the borrow checker. The borrow checker verifies the ownership model and decides if a value in memory (stack or heap) is out of scope or not.
+    - If a value is out of its scope, it’s not accessible to other parts of the program unless it’s borrowed.
+    - When you understand ownership, you’ll have a solid foundation for understanding the features that make Rust unique.
+    - The Ownership Rules
+      - In Rust, every variable owns the value it’s initialized with, and there can only be one owner.
```

Comment:
This is a copy paste https://www.makeuseof.com/rust-ownership/#:~:text=Rust%20verifies%20the%20ownership%20rules,the%20program%20unless%20it's%20borrowed.